app/connectors/httpParsers/SubmittedDividendsHttpParser.scala

```diff
@@ -16,36 +16,30 @@
 
 package connectors.httpParsers
 
-import models.SubmittedDividendsModel
-import play.api.http.Status.{NOT_FOUND, OK, SERVICE_UNAVAILABLE}
+import models.{DesErrorBodyModel, DesErrorModel, SubmittedDividendsModel}
+import play.api.http.Status._
 import uk.gov.hmrc.http.{HttpReads, HttpResponse}
 
 object SubmittedDividendsHttpParser {
-  type SubmittedDividendsResponse = Either[SubmittedDividendsException, SubmittedDividendsModel]
+  type SubmittedDividendsResponse = Either[DesErrorModel, SubmittedDividendsModel]
 
   implicit object SubmittedDividendsHttpReads extends HttpReads[SubmittedDividendsResponse] {
     override def read(method: String, url: String, response: HttpResponse): SubmittedDividendsResponse = {
       response.status match {
         case OK => response.json.validate[SubmittedDividendsModel].fold[SubmittedDividendsResponse](
           jsonErrors =>
-            Left(SubmittedDividendsInvalidJsonException),
+            Left(DesErrorModel(OK, DesErrorBodyModel.parsingError)),
```

Comment:
should this be returning an OK?

Reply:
I'm not sure, probably not but I'm not sure what status it should return in this instance. Might need to be a team discussion